Makefile

```diff
@@ -58,9 +59,9 @@ test/unit:
 test/e2e:
 	$(CRI_BIN) run --rm \
 	           --volume $(CURDIR):$(CURDIR):Z \
-	           --volume $(HOME)/.kube:/root/.kube:Z \
+	           --volume $(subst %/,%,$(dir $(KUBECONFIG))):/root/.kube:Z \
 	           --workdir $(CURDIR) \
-	           -e KUBECONFIG=/root/.kube/config \
+	           -e KUBECONFIG=/root/.kube/$(notdir $(KUBECONFIG)) \
```

Comment:
IIUC the new command is the equivalent of basename [1].
Please consider using the output of basename.

[1] https://man7.org/linux/man-pages/man1/basename.1.html

Reply:
again - this doesn't work:
with notdir:
```
           -e KUBECONFIG=/root/.kube/kubeconfig \
```
with basename:
```
           -e KUBECONFIG=/root/.kube//home/ralavi/.kube/kubeconfig \

```

Reply:
This snippet works:
```suggestion
	           -e KUBECONFIG=/root/.kube/$(shell basename $(KUBECONFIG)) \
```